tests/installation/installation_overview.pm

```diff
@@ -21,7 +21,7 @@ sub run() {
 
     # overview-generation
     # this is almost impossible to check for real
-    assert_screen "inst-overview";
+    assert_screen "installation-settings-overview-loaded";
```

Comment:
As stated in last comment, for sle we use "inst-overview" in many places and it also matches just that we are on "Installation settings" page, but we don't assert that there is content in overview screen. New needle checks that button "next" is changes to "install", as it happens when overview is loaded. On TW and Leap with same needle we match content of the overview, so it's not an issue there. Newly created tag verifies more than "inst-overview" tag, as expects it's initial state.